test/e2e/framework.go

```diff
@@ -383,6 +385,71 @@ func (data *TestData) waitForCRReadyOrDeleted(timeout time.Duration, cr string,
 	return nil
 }
 
+func (data *TestData) getCRProperties(timeout time.Duration, cr_type, cr_name, namespace, key string) (string, error) {
+	value := ""
+	err := wait.Poll(1*time.Second, timeout, func() (bool, error) {
+		cmd := fmt.Sprintf("kubectl get %s %s -n %s -o yaml | grep %s", cr_type, cr_name, namespace, key)
+		log.Printf("%s", cmd)
```

Comment:
I suggest we should use the same logger with operator code, as we do in NCP colibri test. This way is good for log collection as the same method, Moreover, operator logger can support log level setting.

Reply:
it seems to be a common suggestions for all the code in e2e testing, i only used this format as other methods are all using log this way. I'd suggest to separate this suggestion to another patch that we should discuss a unified way to write loggings.